### DISCUSSION TITLE
Can you please clarify something?

### DISCUSSION THREAD

**Discussion by shawwellpete** [matt-FFFFFF/hassio-addon-repository#26] [General]
Hi gusy, I need to move Teslamate from docker/portainer to a home assistant add on. I thought I would start by getting it working under HA OS first. Following the instructions here [https://github.com/.../hass.../blob/main/teslamate/README.md](https://github.com/matt-FFFFFF/hassio-addon-repository/blob/main/teslamate/README.md?fbclid=IwAR3yPKUPknFiATP2PdNTKSwKNFe7AohAMUgeLy__-yecFVaBZCm4xaLcX90)

After starting postgresQL I need to "configure a PostgreSQL data source as follows:" with this image
Any idea where I find that?
![2022-12-13_18-00-37](https://user-images.githubusercontent.com/74064242/207536029-b93a834e-9a4f-4ffc-8798-ffe2c141e85b.jpg)

**Comment by mogs31** [comment]
Hello, I hope you already find the answer ! 
For those who are looking for the information, it has to be setup within grafana --> New datasource (type postgre)
